shell_completions.md

```diff
@@ -173,6 +173,21 @@ backend        frontend       database
 Note that without declaring `rc` as an alias, the completion algorithm would not know to show the list of
 replication controllers following `rc`.
 
+#### Disable completion descriptions
```

Comment:
Could you move this new section under the existing "Descriptions for completions" section?